xmtp/xmtp-react-native
# Revert yarn.lock update from #239 in example

https://github.com/xmtp/xmtp-react-native/pull/239 upgraded dependencies (`yarn.lock`) in sdk folder and example directory. The `yarn.lock` upgraded dependencies in the example directory broke the example app on iOS. 

We should revert the yarn.lock dependencies updates in the example directory until we can get the updates working with the ios example app build. 

<img width="989" alt="image" src="https://github.com/xmtp/xmtp-react-native/assets/1103838/881d16f6-6108-4306-8bdf-df0aec5d981e">


## To test

Before this change latest `beta`

1. fresh clone of xmtp-react-native or `git clean -fdx`
2. run ios see error from screen shot

After this change => should build on a fresh clone
